README_DEVELOPERS.md

```diff
@@ -43,3 +43,25 @@ npm run test -- -- -t react
 ## Replace etalon
 
 To replace etalon image just remove it from `packages/devextreme-cli/testing/__tests__/__image_snapshots__` folder and run tests again
+
+# Modify an Application Template
+
+If you need to modify an application template, we recommend that you generate a real application based on this template and modify this application instead. This is easier because a real application can be run, and you can see how your modifications affect it, which would not be possible if you were to modify the template directly.
```

Comment:
This is written in first person. We usually don't write help docs in first person. Did you have permission to do this? Also, the second sentence is too long with too many ideas. Break it up and rewrite.

Reply:
```suggestion
It is not recommended that you modify an application template directly. Instead, generate a real application based on this template and modify this application. This is easier because a real application can be run, and you can see how your modifications affect it. Follow these instructions:
```

Reply:
Modifying an application template directly is not recommended.